packages/config/config/devices/0x0176/tzwp-102.json

```
@@ -93,6 +93,76 @@
 					"value": 0
 				}
 			]
+		},
+		{
+			"#": "17",
+			"label": "Report on change of power consumption (%)",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 10,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "101",
+			"label": "Power (W) report interval",
+			"valueSize": 4,
```

```suggestion
			"valueSize": 4,
			"unit": "seconds",
```
Do the same for the following parameters please.